executor/trace_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor_test
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/util/testkit"
+)
+
+type testTraceExec struct{}
+
+func (s *testTraceExec) SetupSuite(c *C) {
+
+}
+
+func (s *testSuite) TestTraceExec(c *C) {
```

Comment:
Seems this test file does nothing, can we remove the whole file and add it when adding ut.
And I think it's better to add these integration test in `mysql-test`, not in the ut.

Reply:
Yes. I leave a todo and I will soon add the actual test in another PR.